dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryListDTO.java

```diff
@@ -0,0 +1,111 @@
+package com.dotcms.rest.api.v1.categories;
+
+import java.util.Date;
+
+public class CategoryListDTO {
```

Comment:
add doc

Reply:
This has been addressed in the recent commit